packages/app/src/components/PageEditor/CommentMentionHelper.ts

```diff
@@ -0,0 +1,52 @@
+import { apiv3Get } from '~/client/util/apiv3-client';
+
+export default class CommentMentionHelper {
+
+  editor;
+
+  pattern: RegExp;
+
+
+  constructor(editor) {
+    this.editor = editor;
+    this.pattern = /@[A-Za-z0-9._-]{1,}/;
```

Comment:
Please make this pattern variable a module variable (outside of the class).

Reply:
The pattern already defined outside of the class